googletest/include/gtest/internal/gtest-param-util.h

```diff
@@ -396,6 +397,30 @@ typename ParamNameGenFunc<ParamType>::Type *GetParamNameGen() {
   return DefaultParamName;
 }
```

Comment:
We can delete these now, all 3 
ParamNameGenFunctor GetParamNameGen...
struct ParamNameGenFunc...
typename ParamNameGenFunc<ParamType>....